scipy/stats/tests/test_distributions.py

```diff
@@ -27,13 +27,18 @@
 import scipy.stats.distributions
 
 from scipy.special import xlogy
-from .test_continuous_basic import distcont
+from .test_continuous_basic import distcont, invdistcont
+from .test_discrete_basic import invdistdiscrete
 from scipy.stats._continuous_distns import FitDataError
 from scipy.optimize import root
 
 # python -OO strips docstrings
 DOCSTRINGS_STRIPPED = sys.flags.optimize > 1
 
+# distributions to skip while testing the fix for the support method
+# introduced in gh-13294.
+skip_test_support_gh13294_regression = ['tukeylambda', 'pearson3']
```

Comment:
Maybe indicate why they should be skipped.

Reply:
Done.